NuRadioReco/modules/trigger/radiant_aux_trigger.py

```diff
@@ -0,0 +1,173 @@
+import logging
+import time
+import numpy as np
+from NuRadioReco.framework.trigger import RadiantAUXTrigger
+from NuRadioReco.modules.base.module import register_run
+from NuRadioReco.framework.parameters import channelParameters as chp
+from NuRadioReco.modules.trigger.highLowThreshold import get_majority_logic
+from NuRadioReco.utilities import fft
+import matplotlib.pyplot as plt
+logger = logging.getLogger('radiant_aux_trigger')
+
+def tanh_func(x, b, c):
+    return 0.5*(np.tanh((x-b)/c) + 1)
+
+class triggerSimulator:
+    """
+    """
+
+    def __init__(self):
+        self.__t = 0
+        self.begin()
+
+    def begin(self):
+        return
+    
+    def get_filtered_trace_trigger(self, trace):
+        n_samples = len(trace)
+        spectrum = fft.time2freq(trace, self.sampling_rate)
+        freqs = np.fft.rfftfreq(n_samples, 1 / self.sampling_rate)
+        filtering_out = np.where( (freqs < 0.08) | (freqs > 0.2)) # or!
```

Comment:
In general I would agree. Here, this is the trigger path and the trace is not stored for analysis. I parametrised it this way, so I would prefer to take the hard cut, otherwise I have to fit the trigger curves again.